components/telekonseling/telekonseling-link-section.tsx

```diff
@@ -0,0 +1,25 @@
+import { InternalPageSection } from "~/components/layout/page";
+import StackedLink from "~/components/layout/stacked-link";
+import { Telecounseling } from "~/lib/content/telekonseling";
+
+interface TelecounselingSectionProps {
+  telekonseling: Telecounseling;
+}
+
+export function TelecounselingSection(props: TelecounselingSectionProps) {
+  return (
+    <InternalPageSection className="flex-1 pb-6 space-y-8">
+      {props.telekonseling.supports.map((counseling, index) => (
+        <div key={index} className="space-y-4">
+          <h2 className="text-base font-semibold text-gray-900">
+            {counseling.support}
+          </h2>
+          {/* <p className="text-sm text-gray-500">{counseling.description}</p> */}
```

Comment:
Why do you have this commented code?
If you don't need it, please remove it.

Reply:
I was thinking about how to show this page, wheter put it on the red box or the red line. Will delete it, I think the red box is correct since it represent the whole page.
![telekonseling2](https://user-images.githubusercontent.com/56619123/129510569-16d745bc-1783-4b22-b48c-8960ef8269c6.png)